src/main/java/gtPlusPlus/core/tileentities/general/TileEntityFishTrap.java

```diff
@@ -26,19 +26,19 @@
 public class TileEntityFishTrap extends TileEntity implements ISidedInventory {
 
     private int tickCount = 0;
-    private boolean isInWater = false;
     private final InventoryFishTrap inventoryContents;
     private String customName;
-    private int waterSides = 0;
-    private int baseTickRate = 600 * 5;
+    // The number of water blocks is used as an index to get the tick rate.
+    private final static short[] waterBlocksToTickRate = new short[] { 0, 0, 5600, 4400, 3200, 2000, 1750 };
```

Comment:
6800 is gone and you inserted a completely new value 2000. Why?

Reply:
I got some feedback in the mod channel on the discord to adjust these values. 
Upon reflecting, I will revert these to the intended values and any future re-balancing can be addressed in a separate change with the appropriate people involved in the balance discussion. Apologies for jumping the gun and going out of scope with this specific change.

Reply:
Well I have completely no objection over this trivial balance changes, and it's probably not going to be objected by other people either. However you should clearly state in PR description that you did so, instead of just hiding this change among other things